CMakeLists.txt

```
@@ -641,17 +621,24 @@ add_library (glog
 
 add_library (glog::glog ALIAS glog)
 
+set (glog_libraries_options_for_static_linking)
+
 if (Unwind_FOUND)
   target_link_libraries (glog PRIVATE unwind::unwind)
+  set (glog_libraries_options_for_static_linking "${glog_libraries_options_for_static_linking} -lunwind")
   set (Unwind_DEPENDENCY "find_dependency (Unwind ${Unwind_VERSION})")
 endif (Unwind_FOUND)
 
 if (HAVE_DBGHELP)
   target_link_libraries (glog PRIVATE dbghelp)
+  set (glog_libraries_options_for_static_linking "${glog_libraries_options_for_static_linking} -ldbghelp")
 endif (HAVE_DBGHELP)
 
 if (HAVE_PTHREAD)
   target_link_libraries (glog PRIVATE ${CMAKE_THREAD_LIBS_INIT})
+  if (CMAKE_THREAD_LIBS_INIT)
```

I prefer to leave an empty line before/after conditional blocks for better readability.

```suggestion

  if (CMAKE_THREAD_LIBS_INIT)
```